scripts/openssl.test

```diff
@@ -1228,6 +1233,37 @@ do
 done
 IFS="$OIFS" #restore separator
 
+# Test for RSA-PSS certs interop
```

Comment:
For some reason the CI is unhappy with this change. Any ideas?

```
# Trying to start RSAPSS OpenSSL server on port 61906...
#
#  openssl s_server -accept 61906 -cert "/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/rsapss/server-rsapss.pem" -key "/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/rsapss/server-rsapss-priv.pem" -quiet -CAfile "/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/client-cert.pem" -www -dhparam "/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/dh2048.pem" -verify 10 -verify_return_error -psk 1a2b3c4d -cipher ALL:eNULL -allow_no_dhe_kex
s_server started successfully on port 61906
#
# ./examples/client/client -p 61906 -g  -l ALL -v 4   "-c/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/client-cert.pem" "-k/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/client-key.pem" "-A/tmp/workspace/PRB-fsanitize-addr-v3/wolfssl/scripts/../certs/rsapss/ca-rsapss.pem" -C
wolfSSL error: can't load client cert file, check file and run from wolfSSL home dir
client failed! Suite = ALL version = 4
in cleanup
killing server: OpenSSL_ECDH[E]-ECDSA (59625)
killing server: OpenSSL_RSAPSS (61906)
REGEX_STOP
EXITING SCRIPT with exit code: 1
script returned exit code 1
```

```
# Trying to start RSAPSS OpenSSL server on port 57077...
#
#  openssl s_server -accept 57077 -cert "/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/rsapss/server-rsapss.pem" -key "/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/rsapss/server-rsapss-priv.pem" -quiet -CAfile "/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/client-cert.pem" -www -dhparam "/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/dh2048.pem" -verify 10 -verify_return_error -psk 1a2b3c4d -cipher ALL:eNULL -allow_no_dhe_kex
s_server started successfully on port 57077
#
# ./examples/client/client -p 57077 -g  -l ALL -v 4   "-c/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/client-cert.pem" "-k/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/client-key.pem" "-A/tmp/workspace/PRB-generic-config-parser/wolfssl/./scripts/../certs/rsapss/ca-rsapss.pem" -C
wolfSSL error: can't load client cert file, check file and run from wolfSSL home dir
client failed! Suite = ALL version = 4
in cleanup
killing server: OpenSSL_ECDH[E]-ECDSA (63686)
killing server: wolfSSL_ECDH[E]-ECDSA (55634)
killing server: OpenSSL_RSAPSS (57077)
FAIL scripts/openssl.test (exit status: 1)
```

Reply:
Thanks David. I think it needs a check around if RSA is available before attempting use of RSA-PSS certificates. Am working on an update for it.